codec-http2/src/main/java/io/netty/handler/codec/http2/Http2FlowController.java

```diff
@@ -22,6 +22,12 @@
 public interface Http2FlowController {
 
     /**
+     * Gets the {@link Http2Connection.PropertyKey} used for accessing the {@link Http2Stream.FlowControlState} property
+     * that this controller associates with each {@link Http2Stream}.
+     */
+    Http2Connection.PropertyKey stateKey();
```

Comment:
@Scottmitch I'm not sure I agree.  The flow controller always had to put the state in the property map, the only thing that changed is which method they call.  I see no reason that the flow controller needs to call any other methods than the get/setProperty.  The fact that the flow control state accessors exist in the `Http2Stream` interface is only to enable advanced use cases where the application wants control over flow control windows.